.github/workflows/lint-pr.yml

```diff
@@ -0,0 +1,13 @@
+name: Lint PR
```

Comment:
Judging by the name of the `build-lint-test` workflow, doesn't that one also take care of the linting?

Reply:
linting here is running `npm run lint` in the repo https://github.com/cloudscape-design/actions/blob/main/.github/workflows/build-lint-test.yml#L66

`lint-pr` is linting the PR title to check it matches our [commit convention](https://github.com/cloudscape-design/actions/blob/main/.github/workflows/lint-pr.yml)
we use PT titles to create release notes in the repo so that's why we lint it separately, since we want to call the workflow on PR title changes

Reply:
also this workflow is available in all of our other repos, it was just missed to be added here. 
e.g. https://github.com/cloudscape-design/components/blob/main/.github/workflows/lint-pr.yml